posts/2023-09-01-electing-new-project-directors.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: "Electing New Project Directors"
+author: Leadership Council
+team: Leadership Council <https://www.rust-lang.org/governance/teams/leadership-council>
+---
+
+Today we are launching the process to elect new Project Directors to the Rust Foundation Board of Directors.
+As we begin the process, we wanted to spend some time explaining the goals and procedures we will follow.
+We will summarize everything here, but if you would like to you can read the [official process documentation][pde-process].
+
+We ask all project members to begin working with their Leadership Council representative to nominate potential Project Directors. See the [Candidate Gathering](#candidate-gathering) section for more details. Nominations are due by September 15, 2023.
+
+<!-- TODO: Once we've merged this PR we should link to the final location instead. -->
+[pde-process]: https://github.com/rust-lang/leadership-council/pull/12
+
+## What are Project Directors?
+
+The Rust Foundation Board of Directors has five seats reserved for Project Directors.

```

Comment:
Some context that people reading this might appreciate is how many seats total there currently are on the council (iirc it depends on how many corporate members at the top level?)

I'm willing to be overridden if others think this isn't relevant here.

Reply:
The project directors have a fixed 5 seats. There's no automatic growth with growth of the overall board, but any vote requires a majority of both project & corporate directors, so the growth doesn't dilute vote power for the project.

I'm not sure getting into this level of nuance is hugely helpful here - it's usually a few hoop jumps for folks to understand but gets resolved quickly in comment threads etc. But I'm not opposed.

Reply:
I think at least saying something like "we're rotating out two of the five project director seats" would be helpful and not too much detail. I'll add something about this.

Reply:
Oh, I guess I already had the number of project directors already in there..

I think I'd rather not have the extra detail about total number of seats, growth of the board, and the way voting power is allocated. The whole thing seems like more detail than is really appropriate for this post and anything less than that seems like not enough to stand on its own.